src/components/graph/nodes/network-modification-node.js

```diff
@@ -11,37 +11,70 @@ import makeStyles from '@mui/styles/makeStyles';
 import { useSelector } from 'react-redux';
 import CircularProgress from '@mui/material/CircularProgress';
 import VisibilityIcon from '@mui/icons-material/Visibility';
-import BuildIcon from '@mui/icons-material/Build';
+import VisibilityOffIcon from '@mui/icons-material/VisibilityOff';
 import LockIcon from '@mui/icons-material/Lock';
-import clsx from 'clsx';
+import Tooltip from '@mui/material/Tooltip';
 
 const useStyles = makeStyles((theme) => ({
-    networkModificationSelected: {
+    networkModificationVisualized: {
+        position: 'relative',
         variant: 'contained',
         background: theme.palette.primary.main,
         textTransform: 'none',
         color: theme.palette.primary.contrastText,
         '&:hover': {
             background: theme.palette.primary.main,
         },
+        overflow: 'hidden',
     },
     networkModification: {
-        variant: 'outlined',
-        background: theme.palette.primary.light,
+        background: '#c5c8d1',
         textTransform: 'none',
         color: theme.palette.primary.contrastText,
         '&:hover': {
             background: theme.palette.primary.main,
         },
+        overflow: 'hidden',
+    },
+    outOfBoundIcons: {
+        display: 'flex',
+        flexDirection: 'column',
+        position: 'absolute',
+        right: '-30px',
+        top: '18px',
     },
     buildStatusOk: {
         color: 'green',
     },
     buildStatusInvalid: {
         color: 'indianred',
     },
-    margin: {
-        marginLeft: '10px',
+    buildBannerOK: {
+        display: 'flex',
+        height: '100%',
+        width: '15%',
+        position: 'absolute',
+        top: '0px',
+        left: '0px',
+        background: '#74a358',
```

Comment:
Why not use the same green as in the minimap ?
#74a358 vs #70d136

Also, maybe use properties for the colors, instead of direct values.
This way, we only need to change the properties in one place to adapt the colors of the whole tree.

Reply:
There is also the possibility to use the theme colors here. This way, the contrast is correct in dark and light mode.

`background: theme.palette.success.main,`
works fine here.

Reply:
I created constants to hold the banners background color.
We've carefully chosen the green tint thus I think it's best to keep the same from one theme to another